be/src/exec/pipeline/scan/scan_operator.cpp

```diff
@@ -431,6 +432,8 @@ pipeline::OpFactories decompose_scan_node_to_pipeline(std::shared_ptr<ScanOperat
                 std::make_shared<pipeline::LimitOperatorFactory>(context->next_operator_id(), scan_node->id(), limit));
     }
 
+    ops.emplace_back(std::make_shared<ChunkAccumulateOperatorFactory>(context->next_operator_id(), scan_node->id()));
```

Comment:
should only add ChunkAccumulateOperator when there are non-push-down-filters or runtime filters.

Reply:
some rf only can push down to exchange nodes, so I think we should add this operator after exchange nodes.  from the code,  if all the chunks are great than 0.75*4096, this operator is work in the pass-through style, so I think we can interpolate this operator in the pipeline more aggressively. Another thing is, if the chunk contain BinaryColumn but num_rows is below the 0.75*4096, chunk::append operations are heavy, so can we add a watermark in bytes for chunk contains binary column?